tests/helpers/providers/record_provider.py

```diff
@@ -51,7 +51,23 @@ def author_record():
             "_collections": ["Authors"],
         }
 
-    def record(self, record_type, data=None, with_control_number=False):
+    @staticmethod
+    def add_citations(citation_records=[]):
+        data = []
+        for record in citation_records:
+            data.append(
+                {"record": {"$ref": f"http://localhost:5000/api/literature/{record}"}}
+            )
+        return {"references": data}
```

Comment:
and return only `data` here.

Reply:
Value was changed, now it's not misleading.